book/1-pre-project-phase/data_risk_uncertainties.md

```diff
@@ -0,0 +1,36 @@
+# What are the potential data risks/uncertainties that the AI engineer will be inheriting from the project?
```

Comment:
I suggest to reframe this as "What are the key areas to look out for in the data when framing the AI project?"